extensions/topic_view_extension.rb

```diff
@@ -19,6 +19,26 @@ def self.prepended(base)
       end
     end
 
+    def next_page
+      return super if !topic.is_post_voting?
+      return super if @filter == TopicView::ACTIVITY_FILTER
+
+      @highest_post_number = @filtered_posts.last.post_number
+
+      if @posts.blank?
+        @last_post = nil
+      else
+        @last_post ||= @posts.last
+      end
```

Comment:
Do you need this or can you reuse `last_post`, like the core implementation? https://github.com/discourse/discourse/blob/140b2118af26bbf013bde31fa37ce1c0a62c77b7/lib/topic_view.rb#L188-L202